src/renderer/theme/default.css

```diff
@@ -61,6 +61,7 @@
   --divider: rgba(69, 69, 137, 0.06);
 
   /* BACKGROUND */
+  --background-default: #f8f8fa;
```

Comment:
Added default background color, so we won't be afraid of opacity that might play in